Typeracer/Controllers/GraphController.cs

```diff
@@ -17,27 +13,44 @@
 [Route("api/[controller]")]
 public class GraphController : ControllerBase
 {
+    public Game? Game { get; set; }
+    public Paragraph? Paragraph { get; set; }
+    private readonly AppDbContext _dbContext;
+
+    public GraphController(AppDbContext context)
+    {
+        _dbContext = context;
+    }
+    
+    
     [HttpPost("generate")]
-    public IActionResult GenerateGraph([FromBody] string gameId, AppDbContext context)
+    public IActionResult GenerateGraph([FromBody] string gameId)
```

Comment:
I changed the controller to handle only HTTP requests in my branch... Conflicts incoming!